src/Microsoft.ServiceFabric.Services.Remoting/V1/Client/ServiceProxyFactory.cs

```diff
@@ -45,6 +45,11 @@ internal class ServiceProxyFactory : IServiceProxyFactory
             this.retrySettings = retrySettings;
         }
 
+        public TServiceInterface CreateNonIServiceProxy<TServiceInterface>(Uri serviceUri, ServicePartitionKey partitionKey = null, TargetReplicaSelector targetReplicaSelector = TargetReplicaSelector.Default, string listenerName = null)
+        {
+            throw new NotImplementedException("CreateNonIServiceProxy is not supported in remoting V1");

```

Comment:
Please change the exception type to NotSupportedException and
add the string in the Resource String list. Here is an example on it :  [](url)

Reply:
Sure, no problem.